clients/extension/src/test/utils/webpack.config.js

```diff
@@ -6,7 +6,7 @@
 const path = require("path");
 const glob = require("glob");
 const webpack = require("webpack");
-const raw = require("@bentley/config-loader/lib/IModelJsConfig").IModelJsConfig.init(true /*suppress error*/, true);
+require("@bentley/config-loader").loadEnv(path.join(__dirname, "..", "..", "..", ".env"));
```

Comment:
also, your description says it should be in `imodeljs-config/.env`, am I missing something here?

Reply:
The `imodeljs-config/.env` holds the initial file and then it is copied as a peer of each `package.json` that needs it which is the relative path used here.

The path is a bit of a pain, my one goal was to make it explicit where the file was coming from instead of potentially finding it in a random folder.

One potential enhancement to this is to pass the .env file path in the cmdline when invoking the webpack command.  Similar to the examples [here](https://github.com/motdotla/dotenv#preload).  That would remove the need for the messy path but I do need to check if it works with dotenv-expand as well.

Reply:
Looks like dotenv-expand has yet to merge their support for cli use, https://github.com/motdotla/dotenv-expand/pull/31. Once that's released we could switch to that and completely remove the `loadEnv` method and `@bentley/config-loader` package.